components/public-api/gitpod/experimental/v1/ide_workspace.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1";
+
+service IDEWorkspaceService {
+    // SendHeartbeat sends a heartbeat signal to a running workspace.
+    rpc SendHeartbeat(SendHeartbeatRequest) returns (SendHeartbeatResponse) {}
+
+    // SendClose sends a close signal to a running workspace.
+    rpc SendClose(SendCloseRequest) returns (SendCloseResponse) {}
```

Comment:
As a user of the API, how do I understand the difference between StopWorkspace and SendClose?

Could we fold this operation into SendHeartbeat?

Reply:
Maybe we rename a service to `IDEClientService` to clarify that it is related to a lifecycle of IDE client.

An idea was to separate functions because they are bound to different client lifecycle hooks. I guess initial design was rather about convenience of the implementor?